erpnext/projects/doctype/task/task.py

```diff
@@ -244,6 +244,8 @@ def update_status(self):
 			from datetime import datetime
 			if self.exp_end_date < datetime.now().date():
 				self.db_set('status', 'Overdue', update_modified=False)
+				for project in self.projects:
```

Comment:
This is incorrect. 

Only the Default Project status must be set as overdue.

Reply:
Did this way as the Expected end date set is for the task though it is in multiple projects and status shown in child table are of task.

Reply:
@neilLasrado the expected end date is associated with tasks and not with project, looks good to me. Any reason why only the default project status must be updated?

Reply:
@sahil28297  Neil right here. it should update the status for the default project in the child table. Default Project status associated with the task only. 
let's take e.g for Sprint 1 someone's last updated status is a code review and the second sprint's last updated status is Testing.  if the date elapses then only the default project status needs to change

any way Task status is associated with default project only thats why if you updated default project status it updated task status

Reply:
@MyuddinKhatri please make the necessary changes as suggested by vishal